packages/dx-react-scheduler/docs/reference/day-view.md

```diff
@@ -97,6 +99,7 @@ cellsData | Array&lt;Array&lt;[DayView.CellData](#dayviewcelldata)&gt;&gt; | Spe
 cellComponent | ComponentType&lt;[DayView.DayScaleCellProps](#dayviewdayscalecellprops)&gt; | A component that renders a day scale cell.
 rowComponent | ComponentType&lt;[DayView.RowProps](#dayviewrowprops)&gt; | A component that renders a day scale row.
 groupingPanelComponent? | ComponentType&lt;[GroupingPanel.HorizontalLayoutProps](./grouping-panel.md/#groupingpanelhorizontallayoutprops)&gt; | A component that renders the grouping panel.
+groupedByDate? | boolean | Indicates whether grouping by date is enabled.
```

Comment:
```suggestion
groupedByDate? | boolean | Indicates whether grouping by date is enabled. Takes its value from the `groupByDate` property of the [GroupingState]() plugin.
```

Reply:
Please add a link to `GroupingState` and make the same change to every `groupedByDate` description.